boards/posix/native_posix/native_posix.dts

```diff
@@ -20,7 +20,6 @@
 		zephyr,flash = &flash0;
 		zephyr,entropy = &rng;
 		zephyr,flash-controller = &flashcontroller0;
-		zephyr,ec-host-interface = &hcp;
```

Comment:
Typos in commit message: chagne = change, definding = defining, simply = simplify

Reply:
Thanks, done.